From f6ffec6afba868ea6598c2114c6178dacd775cb4 Mon Sep 17 00:00:00 2001
From: Xiaokang, Qin <xiaokang.qin@intel.com>
Date: Sun, 21 Oct 2012 02:48:43 +0800
Subject: Add a flag control that PRIVATE libm can be chosen

Bug: 63396
Category: Device Enablement
Domain: AOSP-bionic
Severity: Medium
Origin: Internal
Upstream-candidate: No

Change-Id: Ieb333c5b8b58e26b5da4f5b40d22a3fc13eb5cd5
Orig-Change-Id: Icb4c07ed1e8c76f1673c0b74efaaa5158842f525
Signed-off-by: Xiaokang, Qin <xiaokang.qin@intel.com>
Reviewed-on: http://android.intel.com:8080/74475
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: lab_cactus <lab_cactus@intel.com>
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 libm/Android.mk |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/libm/Android.mk b/libm/Android.mk
index 4dfb216..d2c0530 100644
--- a/libm/Android.mk
+++ b/libm/Android.mk
@@ -1,3 +1,4 @@
+ifneq ($(USE_PRIVATE_LIBM),true)
 LOCAL_PATH:= $(call my-dir)
 
 # TODO: this comes from from upstream's libc, not libm, but it's an
@@ -288,3 +289,4 @@ LOCAL_ADDITIONAL_DEPENDENCIES := $(LOCAL_PATH)/Android.mk
 LOCAL_SYSTEM_SHARED_LIBRARIES := libc
 LOCAL_WHOLE_STATIC_LIBRARIES := libm
 include $(BUILD_SHARED_LIBRARY)
+endif
-- 
1.7.4.1

